omnibus/files/private-chef-cookbooks/private-chef/templates/default/oc_erchef.config.erb

```diff
@@ -253,6 +253,9 @@
     {data_collector, [
         {root_url,                "<%= node['private_chef']['data_collector']['root_url'] %>"},
         {token,                   "<%= node['private_chef']['data_collector']['token'] %>"},
+        <% if node['private_chef']['data_collector']['log_skipped_actions'] %>
```

Comment:
Can we add a default value for this to `attributes/default.rb` in the reconfigure cookbooks with a comment about what it does?